src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -111,6 +111,7 @@ export const FeatureFeaturedLink: React.FC<FeatureFeaturedLinkProps> = ({
 
       <Flex flexDirection={size === "large" ? ["column", "row"] : "column"}>
         {subtitle && (
+          // @ts-ignore
           <HTML variant="mediumText" html={subtitle} mt={2} flexBasis="50%" />
```

Comment:
🤔 

That's a weird one. I wonder why these type errors popped up?

Reply:
They're legit errors, was gonna tag @dzucconi about it when the PR is out of draft.

Reply:
What's the error? I don't see it locally when I check out this branch